NeuroID/TrackerClassExtensions/UIClassExtensions/UIViewController.swift

```diff
@@ -45,24 +45,28 @@ public extension UIViewController {
     }
 
     @objc internal var neuroScreenName: String {
-        return className
+        return nidClassName
     }
 
     var tracker: NeuroIDTracker? {
-        if ignoreLists.contains(className) { return nil }
-        if self is UINavigationController, className == "UINavigationController" { return nil }
-        if let tracker = NeuroID.trackers[className] {
+        if ignoreLists.contains(nidClassName) { return nil }
+        // Separating these into individual checks for easy debugging or reverting
+        if self.nidClassName == "RNSNavigationController" { return nil }
+        if self.nidClassName == "RNSScreen" { return nil}
+        if self is UINavigationController, nidClassName == "UINavigationController" { return nil }
```

Comment:
Is this the fix for the double registration?

Reply:
@kevinsites yes